### PR TITLE
Synth facts api

### PR DESCRIPTION
**Pull request by ljades**
Here's my work on modifying the HTC API. See my latest commit message (and past ones if necessary) for details.

### CODE REVIEW

File: htc-api/api/htc_api.py [hunk 1]
```diff
@@ -32,6 +32,7 @@
 global pool
 global log
 
+FACTID_DIABETES = 1;
```
Comment (cjduffett) [inline htc-api/api/htc_api.py]: There's no guarantee that diabetes has a fact ID of 1. You should be querying the synth_ma.synth_disease table to figure out what the diseasefp is. Each stat we track has a "stat_name". So, this query may look like:

SELECT diseasefp FROM synth_ma.synth_disease WHERE stat_name = 'diabetes';

**Comment by ljades** [comment]
Just updated the branch. Diseasefps now dynamically loaded for queries.

**Comment by gregquinn2001** [comment]
Please rename the alias for the synth_ma.synth_county_facts table from f_diabetes to facts or something that better reflects the table
Also, the stat retrieved should not be called shr_diabetes but instead should be called pct_diabetes

File: htc-api/api/htc_api.py [hunk 2]
```diff
@@ -226,8 +238,10 @@ def get_synth_counties_stats():
     """Counties in JSON, statistics only synthetic"""
     log.debug("entering get_synth_counties_stats() IP=%s" % get_ip());
     con = get_db_con()
-    sql = "SELECT s.ct_fips, s.ct_name, s.sq_mi, s.pop, CASE WHEN s.pop > 0 THEN s.pop_male / s.pop ELSE 0 END AS pct_male, CASE WHEN s.pop > 0 THEN s.pop_female / s.pop ELSE 0 END AS pct_female, s.pop_sm " \
-      "FROM synth_ma.synth_county_stats s"
+    sql = "SELECT s.ct_fips, s.ct_name, s.sq_mi, s.pop, CASE WHEN s.pop > 0 THEN s.pop_male / s.pop ELSE 0 END AS pct_male, CASE WHEN s.pop > 0 THEN s.pop_female / s.pop ELSE 0 END AS pct_female, s.pop_sm, " \
+        "f_diabetes.rate / 100 as chr_diabetes " \
```
Comment (cjduffett) [inline htc-api/api/htc_api.py]: I will normalize the rate to a value between 0 and 1 when updating records in the database. Please remove the div/100 here and anywhere else you query for rate so we're consistent.

**Comment by ljades** [comment]
New commit, made recommended changes, except changing table alias from f_diabetes. Because the current implementation of the tables requires a new join per disease, these specifically disease-named aliases will be necessary for it.

**Pull request merged**